### ISSUE TITLE
Redis Environment Variable

### ISSUE THREAD

**Issue by danielristic** [seuros/capistrano-sidekiq#226]
Is there a way to have the startup command include the REDIS_PROVIDER environment variable so that one can configure a specific Redis instance to connect sidekiq processes to?

**Comment by danielristic** [comment]
So I figured out that the environment variable I needed to change was REDIS_URL rather than REDIS_PROVIDER (which expects the NAME of an environment variable that stores the url for Redis and not the url itself, and is more useful for Heroku and the like).

Anyways I did not find a proper way to specify an extra environment variable for the commands that start sidekiq in this gem so I used a "trick" that allows you to define the variable at the begining of .bashrc on the remote server explained in the capistrano repositroy [here](https://github.com/capistrano/capistrano/issues/1884).

I'm still wondering if there's a better way to do that with this gem or in capistrano though.
